proto/process/agent.proto

```diff
@@ -1328,6 +1337,77 @@ message ServiceAccount {
 	repeated string tags = 6;
 }
 
+// reference https://github.com/kubernetes/api/blob/v0.21.5/networking/v1/generated.proto#L259
+// Note: ServiceBackendPort (contains port name and number) is flattened.
+message IngressServiceBackend {
+	string serviceName = 1;
+	string portName = 2;
```

Comment:
Just wonder since` port number` and `port name` are both optional, should we use `oneof` here ?
https://github.com/kubernetes/api/blob/v0.21.5/networking/v1/generated.proto#L493-L504

Reply:
No strong opinion but I'd prefer keeping it aligned with what's used upstream in kubernetes/api. I also noticed that `oneof` is not used at all in the existing type definitions in `agent.proto`.